definitions/contested/json/CaseEventToComplexTypes/CaseEventToComplexTypes.json

```diff
@@ -4817,6 +4817,15 @@
     "DisplayContext": "OPTIONAL",
     "FieldShowCondition": "judgeApproval1.docType=\"DRAFT_ORDER\" AND (judgeApproval1.judgeDecision=\"READY_TO_BE_SEALED\" OR judgeApproval1.judgeDecision=\"JUDGE_NEEDS_TO_MAKE_CHANGES\")"
   },
+  {
+    "LiveFrom": "28/05/2025",
+    "ID": "FR_approveOrdersToJudgeApproval1",
+    "CaseEventID": "FR_approveOrders",
+    "CaseFieldID": "judgeApproval1",
```

Comment:
what about the other judgeApproval fields?

Reply:
No not needed, I thought Id remove this in place of the extraReportFieldsInput